C#/C#.sublime-syntax

```diff
@@ -1422,19 +1435,31 @@ contexts:
 
   attribute_arguments:
     - meta_content_scope: meta.annotation.cs meta.group.cs
+    - match: \)
+      scope: punctuation.section.group.end.cs
     - match: ''
       set: attribute_in
 
   constructor_arguments:
     - meta_content_scope: meta.instance.cs meta.group.cs
-    - match: '(?=[^\s{])'
+    - match: \)
+      scope: punctuation.section.group.end.cs
+      set: maybe_constructor_initializer
+    - include: maybe_constructor_initializer
+
+  maybe_constructor_initializer:
+    - meta_content_scope: meta.instance.cs meta.group.cs
+    - match: (?=[^\s{])
       pop: true
-    - match: '\{'
+    - match: \{
       scope: punctuation.section.braces.begin.cs
       set: initializer_constructor
 
   function_call_arguments:
     - meta_content_scope: meta.function-call.cs meta.group.cs
+    - match: '\)'
```

Comment:
I don't want to be nagging, but it's funny to see you removing quotes in line 1454 while adding a quoted pattern here. ;)

Reply:
it's not nagging, you're quite right that I'm trying to remove quotes in general and somehow let them slip in there :D I guess at some point a clean-up PR to rename contexts to ensure they match our style guidelines and remove unnecessary quotes is in order ;)